marklogic-data-hub/src/test/java/com/marklogic/hub/dataservices/savedQueries/SavedQueriesServiceTest.java

```diff
@@ -0,0 +1,192 @@
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration(classes = ApplicationConfig.class)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class SavedQueriesServiceTest extends HubTestBase {
+
+    private static JsonNode queryDoc;
+    private SavedQueriesService savedQueriesService;
+
+    @BeforeAll
+    void createSavedQueriesServiceInstance() {
+        this.savedQueriesService = SavedQueriesService.on(adminHubConfig.newFinalClient(null));
+    }
+
+    @BeforeEach
+    void before() throws IOException {
+        String jsonString = "{\n" +
+                "  \"savedQuery\": {\n" +
+                "    \"id\": \"\",\n" +
+                "    \"name\": \"some-query\",\n" +
+                "    \"description\": \"some-query-description\",\n" +
+                "    \"query\": {\n" +
+                "      \"searchText\": \"some-string\",\n" +
+                "      \"entityTypeIds\": [\n" +
+                "        \"Entity1\"\n" +
+                "      ],\n" +
+                "      \"selectedFacets\": {\n" +
+                "        \"Collection\": {\n" +
+                "          \"dataType\": \"string\",\n" +
+                "          \"stringValues\": [\n" +
+                "            \"Entity1\",\n" +
+                "            \"Collection1\"\n" +
+                "          ]\n" +
+                "        },\n" +
+                "        \"facet1\": {\n" +
+                "          \"dataType\": \"decimal\",\n" +
+                "          \"rangeValues\": {\n" +
+                "            \"lowerBound\": \"2.5\",\n" +
+                "            \"upperBound\": \"15\"\n" +
+                "          }\n" +
+                "        },\n" +
+                "        \"facet2\": {\n" +
+                "          \"dataType\": \"dateTime\",\n" +
+                "          \"rangeValues\": {\n" +
+                "            \"lowerBound\": \"2020-01-01T13:06:17\",\n" +
+                "            \"upperBound\": \"2020-01-22T13:06:17\"\n" +
+                "          }\n" +
+                "        }\n" +
+                "      }\n" +
+                "    },\n" +
+                "    \"propertiesToDisplay\": [\"facet1\", \"EntityTypeProperty1\"]\n" +
+                "  }\n" +
+                "}";
+        ObjectMapper mapper = new ObjectMapper();
+        queryDoc = mapper.readTree(jsonString);
+    }
+
+    @Test
+    void testSaveAndModifyQuery() {
+        JsonNode firstResponse = savedQueriesService.saveSavedQuery(queryDoc);
+
+        Assertions.assertNotNull(firstResponse);
```

Comment:
I recommend adding this so that this reads a bit easier; it's also consistent with most of our existing tests (unfortunately not all of them):

    import static org.junit.jupiter.api.Assertions.*;